bims/static/js/views/olmap.js

```diff
@@ -61,6 +61,7 @@ define([
             Shared.Dispatcher.on('map:addHighlightPinnedFeature', this.addHighlightPinnedFeature, this);
             Shared.Dispatcher.on('map:switchHighlightPinned', this.switchHighlightPinned, this);
             Shared.Dispatcher.on('map:closeHighlightPinned', this.closeHighlightPinned, this);
+            Shared.Dispatcher.on('map:resetMap', this.resetMap, this);
```

Comment:
i think the best name is refetchingRecords
because basically it is refetch the records